pkg/reconciler/newreconciler/reconciler.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package newreconciler
```

Comment:
Does this need to be its own package because it conflicts with knativeserving_controller.go? Having a hard time seeing what will conflict if we put this in that same knativeserving package.

Reply:
We will move it back to reconciler, after the current model is removed.
If we add this reconciler.go to the current reconciler package, there will be cyclic import issue, so we tentatively create a new package for now. 
When it is time to remove current controller.go and add_knativeserving.go, we will be ok to change this newreconciler back to  reconciler.